lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -985,6 +985,20 @@ describe('modules/platform/bitbucket/index', () => {
 
       expect(await bitbucket.getPr(5)).toMatchSnapshot();
     });
+
+    it('hasReviewers', async () => {
+      const reviewer = {
+        display_name: 'Jane Smith',
+        uuid: '{90b6646d-1724-4a64-9fd9-539515fe94e9}',
+        account_id: '456',
+      };
+      const scope = await initRepoMock();
+      scope.get('/2.0/repositories/some/repo/pullrequests/5').reply(200, {
+        ...pr,
+        reviewers: [reviewer],
+      });
+      expect(await bitbucket.getPr(5)).toMatchSnapshot();
```

Comment:
use `toEqual` instead

Reply:
is that a new way how to do it?  I was following the pattern of how the others are written.

Reply:
Yes, we're trying to avoid snapshots where possible so that results are more readable + faster

Reply:
ok, updated also on other places.